### DISCUSSION TITLE
Topaz training, coordinates in particle.star not recognized

### DISCUSSION THREAD

**Discussion by davidmbelnap** [tbepler/topaz#96] [Q&A] [answered]
I am using Topaz inside of RELION.  I used RELION's manual and autopicking routines to select several thousand particles.  Those were whittled down using 2D classification and selecting only good classes.  Now I want to use the particles from these good classes to train Topaz.  But when I read in the particles.star file, I get an error "no training particles specified."  What am I doing wrong?

One more question:  RELION allows one to combine micrographs taken at different pixel sizes.  Can I use a combined set within Topaz for training and picking?  I assumed not and was running that way when I encountered the problem in the above paragraph.

David

**Comment by davidmbelnap** [comment]
Further information:  I am using RELION 3.1.1 and Topaz 0.2.5.

**Comment by tbepler** [comment]
Can you share a copy of the particles.star file? It isn't clear exactly what the problem is, but that seems like the place to start.

**Comment by davidmbelnap** [comment]
Tristan,

15 MB file wouldn’t go.  Here is a compressed (via “tar -zcvf DMB-particles.star.tgz particles.star”) file.  Only 2 MB.

David


From: David Michael Belnap ***@***.***>
Date: Thursday, March 25, 2021 at 8:35 AM
To: tbepler/topaz ***@***.***>, tbepler/topaz ***@***.***>
Cc: Author ***@***.***>
Subject: Re: [tbepler/topaz] Topaz training, coordinates in particle.star not recognized (#96)

Thank you.  I’ve attached the particles.star file.

David


From: Tristan Bepler ***@***.***>
Reply-To: tbepler/topaz ***@***.***>
Date: Thursday, March 25, 2021 at 4:53 AM
To: tbepler/topaz ***@***.***>
Cc: David Michael Belnap ***@***.***>, Author ***@***.***>
Subject: Re: [tbepler/topaz] Topaz training, coordinates in particle.star not recognized (#96)


Can you share a copy of the particles.star file? It isn't clear exactly what the problem is, but that seems like the place to start.

**Comment by davidmbelnap** [comment, marked as answer]
I found the problem.  I entered the particles.star file under the "input coordinates" line in the External RELION job page.  If I enter instead under the "input particles" line, then the process worked.  I erroneously assumed I needed to submit under the coordinates line because I wanted the coordinate data.  Apparently the RELION routine extracts the coordinates from the particles file for the Topaz job.

Thank you,

David

**Reply by tbepler** [reply]
Great, glad you found the solution!
